CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ Inspired by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 Note: this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.8.1] - Unreleased
+### Added
+- Option to set Honeybadger `controller` via `log_context` in `HoneyBadger.notify()`.
```

Comment:
Isn't it actually called `honeybadger_grouping:` in the log context? If so, I'd mention that. Maybe
```
If the `log_context` key `honeybadger_grouping:` is set, pass that value to the `controller:` keyword argument of `HoneyBadger.notify`.
```

Reply:
Fixed